src/ResourceManager/DataLakeStore/Commands.DataLakeStore.Test/ScenarioTests/AdlsTests.ps1

```diff
@@ -170,16 +180,116 @@ function Test-DataLakeStoreFirewall
 	}
 }
 
+<#
+.SYNOPSIS
+Tests DataLakeStore Account virtual network rules Lifecycle (Create, Update, Get, List, Delete).
+#>
+function Test-DataLakeStoreVirtualNetwork
+{
+    param
+    (
+        $location
+    )
+
+    if ([string]::IsNullOrEmpty($location))
+    {
+        $location = Get-Location -providerNamespace "Microsoft.CognitiveServices" -resourceType "accounts" -preferredLocation "West US";
+    }
+
+	try
+	{
+		# Creating Account
+		$resourceGroupName = Get-ResourceGroupName
```

Comment:
@lewu-msft it appears this test is failing here:

```
Exception:System.Management.Automation.ParentContainsErrorRecordException: Exception calling "GetAssetName" with "2" argument(s): "The given key was not present in the dictionary."
Details:
Script Stack Trace: at getAssetName, C:\workspace\powershell\src\ResourceManager\DataLakeStore\Commands.DataLakeStore.Test\bin\Debug\Common.ps1: line 381
at Get-ResourceGroupName, C:\workspace\powershell\src\ResourceManager\DataLakeStore\Commands.DataLakeStore.Test\bin\Debug\ScenarioTests\Common.ps1: line 30
at Test-DataLakeStoreVirtualNetwork, C:\workspace\powershell\src\ResourceManager\DataLakeStore\Commands.DataLakeStore.Test\bin\Debug\ScenarioTests\AdlsTests.ps1: line 202
```

Reply:
@cormacpayne  I just made change and passed all live tests locally. But still get this getAssetName error. Will do more research.